modules/openapi-generator/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -1190,7 +1190,12 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
       }
     }
 
-    Invocation.Builder invocationBuilder = target.request().accept(accept);
+    Invocation.Builder invocationBuilder;
+    if (accept != null) {
+      invocationBuilder = target.request().accept(accept);
+      } else {
+      invocationBuilder = target.request();
+      }
```

Comment:
Thanks for the PR. Looks like this line and line 1196 are not indented correctly.

Reply:
@wing328  Hi thank you for the review. I just made the update, can you please take a look. Thank you.